lib/scrapers/utils.py

```diff
@@ -1,10 +1,24 @@
 from tornado import gen
 from tornado.httpclient import AsyncHTTPClient, HTTPError
 import ujson as json
+import re
 
 
 http_client = AsyncHTTPClient()
 
+ssn = re.compile("(\d{3}-?\d{2}-?\d{4}|XXX-XX-XXXX)$")
+ccn = re.compile("(?:\d[ -]*?){13,16}")
+phone = re.compile("(\d{3}-?\d{3}-?\d{4}|XXX-XXX-XXXX)$")
```

Comment:
I got these from some standard credit card, ssn, and phone number filters. Right now, without a lot more parsing, we can't just do >=2 numbers because a) some people use numbers in their email addresses and we can't get rid of email addresses out of the body as of right now, b) dates are included in gmail's standard way of concatenating so those will also break that. 

It's gotta be pretty specific or else we'll throw away too much.